app/src/test/java/org/mozilla/fenix/ext/LogTest.kt

```diff
@@ -10,13 +10,6 @@ import io.mockk.mockkStatic
 import io.mockk.verify
 import org.junit.Before
 import org.junit.Test
-import org.junit.runner.RunWith
-import org.mozilla.fenix.TestApplication
-import org.robolectric.RobolectricTestRunner
-import org.robolectric.annotation.Config
-
-@RunWith(RobolectricTestRunner::class)
-@Config(application = TestApplication::class)
```

Comment:
I like that you did some cleanup in this PR and removed the TestRunner where it is unnecessary as well 👍